### PR TITLE
DOCSUP-36376-task-docs-update

### PR DESCRIPTION
**Pull request by lehasm**
## Description
Updated docs for tasks and task suites.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/client/__init__.py [hunk 1]
```diff
@@ -2564,14 +2568,14 @@ def get_task_suites(self, request: search_requests.TaskSuiteSearchRequest) -> Ge
     @expand('patch')
     @add_headers('client')
     def patch_task_suite(self, task_suite_id: str, patch: task_suite.TaskSuitePatch) -> TaskSuite:
-        """Changes the task suite overlap or priority
+        """Changes task suite parameters in Toloka.
```
Comment (alexandervnuchkov) [inline src/client/__init__.py:2571]: Are we changing parameters here or their values? Maybe it's better to write something like `Changes task suite parameter values in Toloka.`?

File: src/client/__init__.py [hunk 2]
```diff
@@ -2564,14 +2568,14 @@ def get_task_suites(self, request: search_requests.TaskSuiteSearchRequest) -> Ge
...
 
         Args:
-            task_suite_id: ID of the task suite that will be changed.
-            patch: New values.
+            task_suite_id: The ID of the task suite.
+            patch: New parameters.
```
Comment (alexandervnuchkov) [inline src/client/__init__.py:2575]: The same thing as in 2571: `New parameter values.`?

File: src/client/batch_create_results.py [hunk 3]
```diff
@@ -1,83 +1,83 @@
...
+        validation_errors: A list with validation errors in input task suites. The list is filled if the request parameter `skip_invalid_items` is `True`.
+    """
+)
+UserBonusBatchCreateResult = _create_batch_create_result_class_for(
+    UserBonus,
+    """A list with the results of creating rewards for Tolokers.
+
+    Attributes:
+        items: Object with information about issued bonuses.
+        validation_errors: Object with validation errors. Returned if the parameter is used in the request skip_invalid_items=True.
```
Comment (alexandervnuchkov) [inline src/client/batch_create_results.py:72]: Perhaps, use the text from previous methods (The list is filled if the request parameter `skip_invalid_items` is `True`.)?

File: src/client/task.py [hunk 4]
```diff
@@ -129,19 +119,28 @@ class CreateTaskParameters(Parameters):
...
 
 
 @inherit_docstrings
 class CreateTasksParameters(CreateTaskParameters):
-    """Parameters for Tasks creation controlling
+    """Parameters used with the [create_tasks](toloka.client.TolokaClient.create_tasks.md) method.
 
-    Used when creating many Tasks.
+    If the operation is started in an asynchronous mode,
+    we recommend to send the `operation_id` to avoid creating the same tasks multiple times. You can use this ID later to get information about the operation.
```
Comment (alexandervnuchkov) [inline src/client/task.py:137]: `we recommend sending` or `we recommend that you send` is better

File: src/client/task_suite.py [hunk 5]
```diff
@@ -103,20 +98,22 @@ class TaskSuiteCreateRequestParameters(Parameters):
...
-    """Parameters for changing specific TaskSuite
+    """Parameters for changing a task suite.
 
     Attributes:
-        issuing_order_override: The priority of a task suite among other sets in the pool. Defines the order in which
-            task suites are assigned to Tolokers. The larger the parameter value, the higher the priority.
-            This parameter can be used if the pool has issue_task_suites_in_creation_order: true.
-            Allowed values: from -99999.99999 to 99999.99999.
+        issuing_order_override: The priority of a task suite.
+            It influences the order of assigning task suites to Tolokers in pools with the `issue_task_suites_in_creation_order` parameter set to True.
```
Comment (alexandervnuchkov) [inline src/client/task_suite.py:115]: Better "parameter set to `True`" as in the previous methods

File: src/client/__init__.py [hunk 6]
```diff
@@ -2153,34 +2154,33 @@ def create_tasks(
...
         By default, `create_tasks` starts asynchronous operation internally and waits for the completion of it. Do not
-        change `async_mode` to False, if you do not understand clearly why you need it.
+        change `async_mode` to `False`, if you do not understand clearly why you need it.
 
         Args:
-            tasks: List of tasks to be created.
-            parameters: Parameters for Tasks creation controlling. Defaults to None, in which case the asynchronous
-                operations is used.
+            tasks: A list of tasks to be created.
+            parameters: Additional parameters of the request. Default: `None` — asynchronous mode is used.
```
[the remark this answers is not in the document]
Comment (lehasm) [reply src/client/__init__.py]: Hi, @alexdrydew 
`parameters` are substituted with their fields when docs are generated. I think there is no use to describe default values here.
But there were no default values described for those fields. I have added them. Please see the following commit. Do you think that this fix is enough?

By the way, automatic insertion of default values into the docs seems to be the best solution, doesn't it? (We may discuss it later.)

[the remark this answers is not in the document]
Comment (alexdrydew) [reply src/client/__init__.py]: Yes, looks good, thank you

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**